File: src/main/java/utils/StringUtils.java
package utils;

public class StringUtils {
    public static String integerToStringBinary(int integer, int finalLengthOfBinary){
        String binary = Integer.toBinaryString(integer);
        //It´ll prepend n zeros until the string have the size of binary especified on the second parameter
        while(binary.length() != finalLengthOfBinary){
            binary = "0" + binary;
        }
        return binary;
    }

    public static String createStreamOnZeros(int howManyZeros){
        // Enter 3 -> Return 000
        // Enter 7 -> Return 0000000
        return new String(new char[howManyZeros]).replace("\0", "0");
    }

}

File: frontend/public/utils/StringUtils.js
class StringUtils {
    static createStreamOnZeros(quantity) {
        // howManyZeros 3 -> Return 000
        // howManyZeros 7 -> Return 0000000
        return "0".repeat(quantity);
    }

    static createStreamWithOnes(quantity) {
        // howManyZeros 3 -> Return 111
        // howManyZeros 7 -> Return 1111111
        return "1".repeat(quantity);
    }
}

module.exports = StringUtils;
